docs/api/fibre_channel_interfaces.md

```diff
@@ -0,0 +1,43 @@
+
+# nimbleclient.v1.api.fibre_channel_interfaces
+
+
+## FibreChannelInterface
+```python
+FibreChannelInterface(self,
```

Comment:
Again same comment. Declaration should be one liner and consistent with other object sets.